Makefile

```diff
@@ -82,6 +82,12 @@ vulncheck:
 	@go tool govulncheck ./...
 	@echo "Completed vulnerability check!"
 
+nancy:
+	@echo "==================================================================="
+	@echo "Running Nancy vulnerability scanner..."
+	@go list -json -deps ./... | (cd tool && nancy sleuth --exclude-vulnerability-file ../.nancy-ignore)
```

Comment:
would be nice to just use the Docker image so you don't to install it on your machine

```suggestion
	@go list -json -deps ./... | docker run --rm -i \
	  --volume "$(PWD)":/workspace \
	  --workdir /workspace \
	  sonatypecommunity/nancy:latest sleuth \
	  --exclude-vulnerability-file .nancy-ignore
```

Reply:
We can also pin the version here, latest would be `v1.0.51` as of today